src/liquid/components/ld-checkbox/readme.md

```diff
@@ -242,7 +238,9 @@ document.querySelectorAll('#example-indeterminate-1 input, #example-indeterminat
 {% example '{ "background": "light" }' %}
 <ld-checkbox tone="dark"></ld-checkbox>
 
-<ld-checkbox tone="dark" disabled></ld-checkbox>
```

Comment:
Is removing the disabled component from the example intended?

Reply:
Yes, see my comment above ☝️